webapp/src/shared/components/Input/SearchUserForm.tsx

```diff
@@ -0,0 +1,25 @@
+import { Input, InputVariant } from '../';
+import { IconVariant } from '../Icon/Icon';
+import './SearchUserForm.css';
+
+type SearchProps = {
+  search: string;
+  setSearch: React.Dispatch<React.SetStateAction<string>>;
+};
+
+export default function SearchUserForm({ search, setSearch }: SearchProps) {
+  return (
+    <div className="search-user-form">
+      <Input
+        variant={InputVariant.DARK}
+        iconVariant={IconVariant.SEARCH}
+        placeholder="Search"
```

Comment:
the placeholder has lowercase s, according to figma.
'search'  not 'Search'

Reply:
Done